packaging/config/agent.yaml

```diff
@@ -0,0 +1,29 @@
+# Path where Consul auto-loads configuration files: the node meta-data will be stored there as `trento-metadata.json`
+# Legacy, it will be removed in the future
+# defaults to ./consul.d
+
+# consul-config-dir: /srv/consul/consul.d
```

Comment:
More nit-picking: I'd add a double comment  `##` for actual comments and a single one for configuration values. This makes it clearer for the user to quickly identify what the configurable values are.

Reply:
Sure thing! Changed. Thanks for this, I might be overlooking these things, so well done.